eks-hybrid-iam-ra-setup.sh

```diff
@@ -3,14 +3,19 @@
 exec >&2
 set -eu -o pipefail
 
-declare -r SECRETS_DIR="/.bottlerocket/rootfs/root/.aws"
+declare -r HOST_ROOTFS="/.bottlerocket/rootfs"
+declare -r SECRETS_DIR="${HOST_ROOTFS}/root/.aws"
+declare -r AWS_SIGNING_HELPER_UPDATE_SYSTEMD_SERVICE=aws_signing_helper_update.service
+declare -r AWS_SIGNING_HELPER_UPDATE_SYSTEMD_SERVICE_DESTINATION="${HOST_ROOTFS}/run/systemd/system/${AWS_SIGNING_HELPER_UPDATE_SYSTEMD_SERVICE}"
```

Comment:
nit: to cut down some of the verbosity:
```suggestion
declare -r HELPER_SERVICE=aws-signing-helper-update.service
declare -r UNIT_DIR="${HOST_ROOTFS}/run/systemd/system"
```

Reply:
Done